### PR TITLE
24 filtering service

### PR DESCRIPTION
**Pull request by ghostaccount737**

### CODE REVIEW

File: vizrec/package.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (lastlegion) [inline vizrec/package.json]: move the cubejs files to `/query`

**Comment by lastlegion** [comment]
please remove the mongo binary files, use docker instead.

File: query/schema/Orders.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+cube(`Orders`, {
```
Comment (lastlegion) [inline query/schema/Orders.js]: how do we create new schemas for newer datasets?

File: query/schema/Orders.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  sql: `
+  select 1 as id, 100 as amount, 'new' status
+  UNION ALL
+  select 2 as id, 200 as amount, 'new' status
+  UNION ALL
+  select 3 as id, 300 as amount, 'processed' status
```
Comment (lastlegion) [inline query/schema/Orders.js]: please remove this file, our system should be generic enough to create new schemas on the fly. Move this to tests directory and write tests.

File: query/schema/Orders.js (hunk 2 above)
Comment (PVinay737) [reply query/schema/Orders.js]: Working on that

Comment (ghostaccount737) [reply query/schema/Orders.js]: Working on that

**Comment by ghostaccount737** [comment]
Adding clean code in a new branch

**Pull request closed without merging**